backends/gdx-backend-robovm/src/com/badlogic/gdx/backends/iosrobovm/IOSMusic.java

```diff
@@ -58,7 +59,10 @@ public void play () {
 			// OALAudioTrack needs to execute preloadURL() once to store the file path. From then on we avoid
 			// calling it again to avoid instantiating a new AVAudioPlayer every time.
 			if (!initialized) {
-				initialized = track.playFile(filePath);
+				if (looping)
+					initialized = track.playFile(filePath);
+				else
+					initialized = track.playFile(filePath, -1);
```

Comment:
@Berstanio noticed this, not me, but it looks like the -1 might need to be double-checked.

> in the setLooping code it looks like "-1" needs to be set, if there should be looping:
>    ```java
>    @Override
>    public void setLooping (boolean isLooping) {
>        track.setNumberOfLoops(isLooping ? -1 : 0);
>        looping = isLooping;
>    }
> ```
> Do I miss something?

Reply:
Absolutely, thanks!